srcpkgs/asahi-scripts/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'asahi-scripts'
+pkgname=asahi-scripts
+version=20240822
+revision=1
+archs="aarch64*"
+build_style=gnu-makefile
+make_install_args="install-dracut install-mkinitcpio"
+depends="asahi-firmware"
+short_desc="Miscellaneous admin scripts for the Asahi Linux reference distro"
```

Comment:
```suggestion
short_desc="Miscellaneous admin scripts for Apple silicon devices"
```
perhaps? we aren't the asahi reference distro

Reply:
Done